compiler/test/dotty/tools/repl/TabcompleteTests.scala

```diff
@@ -8,7 +8,7 @@ class TabcompleteTests extends ReplTest {
 
   /** Returns the `(<instance completions>, <companion completions>)`*/
   private[this] def tabComplete(src: String)(implicit state: State): List[String] =
-    completions(src.length, src, state).map(_.value)
+    completions(src.length, src, state).map(_.value).sorted
```

Comment:
I think we might as well make `completions` return a sorted result.

Reply:
JLine has its own mechanism to sort completions. So it doesn't if you sort the result